graphql-proxy/src/schema/schema.ts

```diff
@@ -0,0 +1,6 @@
+import dummy from "./dummy/typeDefs";
```

Comment:
Does this file have any purpose? It's identical to `typeDefs.ts`, but this file is not used.